src/System.Text.Primitives/System.Text.Primitives.csproj

```diff
@@ -11,5 +11,6 @@
   <ItemGroup>
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemCompilerServicesUnsafeVersion)" />
+    <PackageReference Include="System.Numerics.Vectors" Version="$(CoreFxVersion)" />
```

Comment:
The CoreFxVersion is hardwired to 4.3.0. You need to add a new tag to tools dependencies.prop for System.Numerics.Vectors and set it to the same value as SystemMemoryVersion (for now). I might have to fix our auto-update tool to update this version number with the other assemblies, but I can do that later.

Once you set the version right and restore, you should have access to the Narrow, Widen, etc.